### DISCUSSION TITLE
Haystack integration

### DISCUSSION THREAD

**Discussion by mhillebrand** [FalkorDB/FalkorDB#857] [Ideas]
I love FalkorDB's integration with LlamaIndex, but have you considered doing the same for [Haystack 2](https://haystack.deepset.ai/)?

**Comment by gkorland** [comment]
@mhillebrand thanks for asking, we would love to see FalkorDB's support in Haystack.
But, as much I can tell haystack doesn't have GraphStore and only support DocumentStore.

**Reply by Hansehart** [reply]
Neo4J e.g is listed as "Vector-capable NoSQL DBs":

https://docs.haystack.deepset.ai/docs/choosing-a-document-store

And you can retrieve documents using cypher:

https://haystack.deepset.ai/integrations/neo4j-document-store
